lib/commands/access.jsx

```diff
@@ -0,0 +1,159 @@
+        'Remove the ability of users and teams to have read-only or ' +
+          'read-write access to a package',
+        Access.options,
+        async argv => accessRevoke(argv)
+      )
+      .command(
+        'ls-packages [<entity>]',
+        'Show all of the packages a user or a team is able to access, along ' +
+          'with the access level, except for read-only public packages',
+        Access.options,
+        async argv => accessLsPackages(argv)
+      )
+      .command(
+        'ls-collaborators [<spec> [<user>]]',
+        'Show all of the access privileges for a package. Will only show ' +
+          'permissions for packages to which you have at least read access. ' +
+          'If <user> is passed in, the list is filtered only to teams that ' +
+          'user happens to belong to',
+        Access.options,
+        async argv => accessLsCollaborators(argv)
+      )
+      .command(
+        'edit <package>',
+        'Set the access privileges for a package at once using $EDITOR',
+        Access.options,
+        async argv => accessEdit(argv)
+      )
+  },
+  options: Object.assign(require('../common-opts.js', {}))
+}
+
+const figgyPudding = require('figgy-pudding')
+const npmConfig = require('../config.js')
+const libnpm = require('libnpm')
+
+const Config = figgyPudding({
+  json: {},
+  loglevel: {},
+  parseable: {},
+  silent: {}
+})
+
+const parseOpts = argv => Config(npmConfig().concat(argv).concat({
+  log: require('npmlog')
+}))
+
+const render = (opts, content) => {
+  const { h, renderToString } = require('ink')
+  const Table = require('ink-table').default
+
+  if (opts.json) {
+    console.log(JSON.stringify(content, null, 2))
+  } else if (opts.parseable) {
+    console.log(['collaborator', 'access'].join('\t'))
+    Object.keys(content).forEach(collab => {
+      console.log([collab, content[collab]].join('\t'))
+    })
+  } else if (!opts.silent && opts.loglevel !== 'silent') {
+    const data = Object.keys(content).map(collab => {
+      return { collab, role: content[collab] }
+    })
+    console.log(renderToString(<Table data={data}/>))
+  }
+}
+
+async function accessPublic (argv) {
+  await libnpm.access.public(argv.spec, parseOpts(argv))
+}
+
+async function accessRestricted (argv) {
+  await libnpm.access.restricted(argv.spec, parseOpts(argv))
+}
+
+async function accessGrant (argv) {
+  await access.grant(argv.spec, argv.team, argv.permissions, parseOpts(argv))
+}
+
+async function accessRevoke (argv) {
+  await libnpm.access.revoke(argv.spec, argv.team, parseOpts(argv))
+}
+
+async function accessLsPackages (argv) {
+  const getPackagesByCurrentUser = () => {
+    // TODO: grab current authenticated user from pending whoami support
+  }
+
+  const entity = argv.entity
+    ? argv.entity
+    : getPackagesByCurrentUser()
+
+  const opts = parseOpts(argv)
+  // TODO: error when opts is used as 2nd param in .lsPackages
+  const packages = await libnpm.access.lsPackages(entity)
```

Comment:
For my education, when using `opts` in the `libnpm.access.lsPackages` call (following other subcommands' pattern for the `libnpm.access.*` call), I ran into an error from lib indicating the expected value for 2nd argument (assuming `opts`) is string or `null`.

The [`access.lsPackages(entity, [opts])` doc](https://www.npmjs.com/package/libnpmaccess#accesslspackagesentity-opts---promise), though, seems to indicate the `opts` parameter would be an object. Would you know why?

```js
const packages = await libnpm.access.lsPackages(entity, opts)
```

```
{ Error: Argument #2: Expected string or null but got object
```

Reply:
that's very strange. It might be something else -- what's the full stack trace?

Reply:
oh nevermind I see where this is happening, one sec.

Reply:
oh nevermind I see where this is happening, one sec.

...try bumping `libnpmaccess` to `3.0.1` and try again.

Reply:
@zkat,
Installed `libnpmaccess` to `3.0.1`at root and passed the test using `opts` when calling `libnpm.access.lsPackages`. Looked good. 👍 

I didn't commit the additional `libnpmaccess` dep at `3.0.1` & the `opt` param to the PR though, since `"libnpm": "^1.3.0"` seems to be "driver" that installs `libnpmaccess`.

Not sure if directly adding `libnpmaccess` dep at `3.0.1` to root's `package.json` would be the right thing to do.

If `libnpmaccess` dep at `3.0.1` to root's package.json is indeed the correct update, I'm happy to push my local edit to the PR. Thanks!